backend/src/main/java/com/woowacourse/gongcheck/core/domain/task/RunningTasks.java

```diff
@@ -17,6 +17,12 @@ public void validateCompletion() {
         }
     }
 
+    public void check() {
```

Comment:
```suggestion
    public void AllCheck() {
```
모두 체크한다라는 의미가 더 명확할 것 같네요 :)

Reply:
오리의 의견에 동의합니당 :)

Reply:
할거라면 checkAll이 더 좋을 것 같아요

Reply:
checkAll로 하겠습니다~!